src/stylesheets/searchBar.scss

```diff
@@ -0,0 +1,156 @@
+@import "variables";
+
+.search-bar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 400px;
+  height: 2rem;
+  border-radius: 8px;
+  border: solid 1px $silver-gray;
+  background-color: $tc-white;
+  padding: 0.3rem 0.5rem;
+
+  &:focus-within {
+    border-color: $mid-gray;
+  }
+
+  .search-input {
+    flex-grow: 1;
+    font-size: $input-font-size;
+    font-family: $default-font-family;
+    background: none;
+    color: $mid-gray;
+    border: none;
+    outline: none;
+    padding: 0 6px;
+  }
+
+  .search-button {
```

Comment:
Can we make the **search button icon** same(similar) size as the other 2 icons in the search bar?
<img width="468" alt="image" src="https://user-images.githubusercontent.com/34193915/83823530-161dbf00-a689-11ea-8466-bc0486d5de50.png">

Reply:
I specified the font size in css for the search icon now... before I think it was using the browser default? I can't really tell if it's fixed cuz my default search icon size looks the same as other icons LOL. Let me know if it's fixed.